CONTRIBUTING.md

```diff
@@ -6,13 +6,58 @@ When submitting a pull request (PR), please use the following guidelines:
   the same coding style as the code that you are modifying. If you are using
   IntelliJ, you can import our code style settings xml:
   [elide-intellij-codestyle.xml](https://github.com/yahoo/elide/raw/master/elide-intellij-codestyle.xml).
-- Do add/update documentation appropriately for the change you are making.
+- Do add/update documentation appropriately for the change you are making.  Documentation for Elide is maintained in [another repo](https://github.com/yahoo/elide-doc).
 - Bugfixes should include a unit test or integration test reproducing the issue.
 - Do not use author tags/information in the code.
 - Always include license header on each file your create. See [this example](https://github.com/yahoo/elide/blob/master/elide-core/src/main/java/com/yahoo/elide/Elide.java)
 - Try to keep pull requests short and submit separate ones for unrelated
   features, but feel free to combine simple bugfixes/tests into one pull request.
 - Keep the number of commits small and combine commits for related changes.
   Each commit should compile on its own and ideally pass tests.
+- Update the [changelog](https://github.com/yahoo/elide/blob/master/changelog.md) with a description of the changes.
 - Keep formatting changes in separate commits to make code reviews easier and
   distinguish them from actual code changes.
+- All code must receive at least one positive review from _another_ committer to be merged to master.
+
+# Community
+
+While anyone is free to open pull requests against the project, Elide has two defined roles:
+- *Committers* - Are the gatekeepers of the repository.  They approve pull requests and have write access to merge against master or other mainline branches.  Committers also help set the direction for the project by deciding on major enhancements or design changes.
```

Comment:
Should we detail a more complete process for committers and admins in this doc? Perhaps a doc solely related to that may make more sense. 

Anyway, what I’m expecting is something outlining the primary communication channels for committees/admins to stay on the same page with design and project movement.

I imagine much of this conversation to take place in a public place, but formalizing a location may make it easier for everyone to stay in sync.